src/resolvers/series.ts

```diff
@@ -32,9 +36,11 @@ export class SeriesResolver {
       JOIN event e on ne.id = e.nft_id
       WHERE e.interaction = 'BUY'
       GROUP BY ce.id, me.image, ce.name 
-      ORDER BY total DESC
-      LIMIT $1 OFFSET $2;
-    `, [limit, offset])
+      ORDER BY $1 $2
```

Comment:
Can we maybe leverage interfaces for this ? 

```ts
enum OrderBy {
  DESC = 'DESC',
  ASC = 'ASC'
}
```

with string we are bit prone to some sql hack

Reply:
🏴‍☠️ Never trust user input